static/app/views/alerts/rules/issue/setupMessagingIntegrationButton.tsx

```diff
@@ -85,13 +86,7 @@ function SetupMessagingIntegrationButton({
       features={integrationProvidersQuery[0].data.providers[0]?.metadata?.features}
     >
       {({disabled, disabledReason}) => (
-        <Tooltip
-          title={
-            disabled
-              ? disabledReason
-              : t('Send alerts to your messaging service. Install the integration now.')
-          }
-        >
+        <div>
```

Comment:
qq: why is this div required?

Reply:
if we don't wrap the button then it looks like this 😭
![image](https://github.com/user-attachments/assets/73d311d4-4b1d-47a4-b9df-fab77e988b4f)